src/import_svg.cc

```diff
@@ -51,8 +54,8 @@ Polygon2d *import_svg(const std::string &filename, const double dpi, const bool
 		double width_mm = 0.0;
 		double height_mm = 0.0;
 
-		dim_t min{1.0/0.0, 1.0/0.0};
-		dim_t max{-1.0/0.0, -1.0/0.0};
+		dim_t min{MAX_DOUBLE, MAX_DOUBLE};
+		dim_t max{MIN_DOUBLE, MIN_DOUBLE};
```

Comment:
Just thinking out loud:
1.0/0.0 looks like a shorthand for infinite to me, not a shorthand for max_double.
But looking at the code, there is no special handling for +infinite or -infinite, so I guess it is assumed that min and max are overwritten anyway, so that both infinite and max_double do the job, but max_double being slightly less error prone then infinity if not overwritten for some reason.

Reply:
Yeah I changed this due to a warning [C4723](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4723?view=vs-2017) I was getting.  And actually with the current build settings `/we4316 /EHsc` the warning doesn't even show anymore when I change back to `1.0/0.0` so I could go either way with that change.  There were so many warnings when I started, I was doing my best to whittle them down.

Reply:
I'm going to change that to Eigen::AlignedBox anyway :)

Reply:
See #2844

Reply:
OK I just reverted my changes in this file to avoid merge conflicts.